src/compositemark/index.ts

```diff
@@ -53,20 +53,18 @@ export function remove(mark: string) {
   delete compositeMarkRegistry[mark];
 }
 
-export type CompositeEncoding<F extends Field = Field> = Encoding<F> & ErrorExtraEncoding<F>;
+export type CompositeEncoding<F extends Field> = Encoding<F> & ErrorExtraEncoding<F>;
```

Comment:
Can you explain why do we remove the default `= Field` here?

Reply:
I found it less confusing to explicitly say the type. It makes me more confident that we didn't miss using string somewhere.

Reply:
Are you okay with this?